apps/tsilhqotin-language-hub/src/Components/Pages/Contact/Contact.tsx

```diff
@@ -0,0 +1,57 @@
+import DevicesIcon from '@mui/icons-material/Devices';
+import LocalPhoneIcon from '@mui/icons-material/LocalPhone';
+import PlaceIcon from '@mui/icons-material/Place';
+import { Divider } from '@mui/material';
+import ScrollToTop from '../../Widgets/ScrollButton/ScrollToTop';
+import './Contact.css';
+import { ContactsDisplay } from './ContactsDisplay';
+import getContactInfos from './getContactInfos';
+
+const contactInfosReadResult = getContactInfos();
+
+export function Contact() {
+    if (contactInfosReadResult instanceof Error)
+        return (
+            <div>
+                <h1>Error Boundary</h1>
+                <p>
+                    This application has encountered the following error:
```

Comment:
One thing that comes to my mind is that we may want to throw the error instead \ as well. This is not the kind of Error we anticipate the user seeing. We should catch that the config is broken before we ship to staging \ production.

The pattern you have used would be a really good way to display 500 errors coming from system errors the back-end encounters.